crates/nu-command/src/filesystem/glob.rs

```diff
@@ -123,7 +123,7 @@ impl Command for Glob {
     }
 
     fn extra_usage(&self) -> &str {
-        r#"For more glob pattern help, please refer to https://github.com/olson-sean-k/wax"#
+        r#"For more glob pattern help, please refer to https://github.com/olson-sean-k/wax/blob/master/README.md#patterns"#
```

Comment:
I'd rather keep the short URL in case the author decides to restructure the README.

Reply:
The old link points to the top of the repo and you have to open readme and read down quite a way to find doc on the pattern and metachars.   
I see  https://docs.rs/wax/latest/wax/ points to `https://github.com/olson-sean-k/wax/blob/master/README.md`, maybe that's a stable enough target?

Reply:
That could work as well :+1:

Reply:
@bobhy Could you still use the shorter docs.rs link?